### PR TITLE
Improve Intakes documentation

### PR DESCRIPTION
**Pull request by gbossert**

### CODE REVIEW

File: docs/integrations/squid.md [hunk 1]
```diff
@@ -1,70 +1,245 @@
 uuid: a0dbb8f3-ca1c-4c6b-aafa-595bd430c0cb
 name: Squid
 
-## Overview
-Squid is a caching proxy for the Web supporting HTTP, HTTPS, FTP, and more. It reduces bandwidth and improves response times by caching and reusing frequently-requested web pages. Squid has extensive access controls and makes a great server accelerator.
+# Squid
+
+Squid is a caching proxy for the Web. It reduces bandwidth and improves response times by caching and reusing frequently-requested web pages. Squid has extensive access controls and makes a great server accelerator. The analysis of squid logs by SEKOIA.IO can detect web threats along as in-progress intrusion and exfiltration attempts.
```
Comment (otetard) [inline docs/integrations/squid.md:6]:
```suggestion
Squid is a caching proxy for the Web. It reduces bandwidth and improves response times by caching and reusing frequently-requested web pages. Squid has extensive access controls and makes a great server accelerator. The analysis of Squid logs by SEKOIA.IO can detect web threats along as in-progress intrusion and exfiltration attempts.
```

File: docs/integrations/squid.md [hunk 2]
```diff
@@ -1,70 +1,245 @@
...
+
+## Data Sources
+
+Following Table lists the data source offered by this integration.
```
Comment (otetard) [inline docs/integrations/squid.md:10]:
```suggestion
Following table lists the data source offered by this integration.
```

File: docs/integrations/squid.md [hunk 3]
```diff
@@ -1,70 +1,245 @@
...
+        "10.16.12.86"
+      ]
+    }
+	
+	```
+
+
+## Configure
+
+As of now, the main solution to collect Squid logs leverages the RSyslog recipe. Please share your experiences with other recipes by editing this documentation.
```
Comment (otetard) [inline docs/integrations/squid.md:210]:
```suggestion
As of now, the main solution to collect Squid logs leverages the Rsyslog recipe. Please share your experiences with other recipes by editing this documentation.
```

File: docs/integrations/squid.md [hunk 4]
```diff
@@ -1,70 +1,245 @@
...
+
+### RSyslog
+
+In this Section, we detail how to configure Squid’s logging output for SEKOIA.IO by means of the RSyslog transport. We hereby focus on the configuration of Squid and invite the reader to the [Rsyslog Transport](../transport/rsyslog/) documentation to forward these logs to SEKOIA.IO.
```
Comment (otetard) [inline docs/integrations/squid.md:214]:
```suggestion
In this Section, we detail how to configure Squid’s logging output for SEKOIA.IO by means of the Rsyslog transport. We hereby focus on the configuration of Squid and invite the reader to the [Rsyslog Transport](../transport/rsyslog/) documentation to forward these logs to SEKOIA.IO.
```

File: docs/integrations/squid.md [hunk 5]
```diff
@@ -1,70 +1,245 @@
...
+    }
+	
+	```
+
+
+## Configure
+
+As of now, the main solution to collect Squid logs leverages the RSyslog recipe. Please share your experiences with other recipes by editing this documentation.
+
+### RSyslog
```
Comment (otetard) [inline docs/integrations/squid.md:212]:
```suggestion
### Rsyslog
```

File: docs/integrations/squid.md [hunk 6]
```diff
@@ -1,70 +1,245 @@
...
+
+In this Section, we detail how to configure Squid’s logging output for SEKOIA.IO by means of the RSyslog transport. We hereby focus on the configuration of Squid and invite the reader to the [Rsyslog Transport](../transport/rsyslog/) documentation to forward these logs to SEKOIA.IO.
+
+To configure Squid logging, you can create a new configuration `99-sekoiaio.conf` file in the `/etc/squid/conf.d/` directory of your server. With most of Squid configurations (including Debian, Red Hat, etc.), this file will automatically be used.
```
Comment (otetard) [inline docs/integrations/squid.md:216]:
```suggestion
To configure Squid logging, you can create a new configuration `99-sekoiaio.conf` file in the `/etc/squid/conf.d/` directory of your server. With most of Squid configurations (including Debian, Red Hat Enterprise Linux, etc.), this file will automatically be used.
```

File: docs/integrations/squid.md [hunk 7]
```diff
@@ -1,70 +1,245 @@
...
-
-```bash
-$ sudo service rsyslog restart
-```
 
-### Enjoy your events
-Go to the [events page](https://app.sekoia.io/operations/events) to watch your incoming events.
+## Parser
 
+The following Table lists all the properties that are analyzed and indexed by the parser.
```
Comment (otetard) [inline docs/integrations/squid.md:235]:
```suggestion
The following table lists all the properties that are analyzed and indexed by the parser.
```

**Review by otetard — COMMENTED** [review]
Nice work!

File: docs/integrations/squid.md (hunk 2 above)
Comment (gbossert) [reply docs/integrations/squid.md:10]: In some academic papers, the words Table, Figure, Section & co are capitalized. You can have a look on this [blog](https://erinwrightwriting.com/reference-tables-and-figures-in-text/) for an interesting discussion on this. I have no idea if it is strictly related to more industrial English.

**Pull request merged**